spec/rspec_enumerables.rb

```diff
@@ -0,0 +1,256 @@
+RSpec.describe 'Enumerable' do
```

Comment:
Adds this line at the top `require_relative '../enumerables'`. It will make it executable.

Reply:
Executable File added, my bad, just missed it